shell/browser/electron_browser_client.h

```diff
@@ -23,6 +23,7 @@
 #include "shell/browser/hid/electron_hid_delegate.h"
 #include "shell/browser/serial/electron_serial_delegate.h"
 #include "third_party/blink/public/mojom/badging/badging.mojom-forward.h"
+#include "third_party/blink/public/mojom/loader/transferrable_url_loader.mojom.h"
```

Comment:
is this actually used in this file?

Reply:
It doesn't look like it to me. I'll try removing it and see what happens.

Reply:
02cc3e4f3a76581070e5f736653b92ddfb939617